packages/skill/scripts/test-mcp.sh

```
@@ -0,0 +1,106 @@
+# Requires: jq (https://jqlang.github.io/jq/) and node >= 18
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)